templates/named.conf.j2

```diff
@@ -40,6 +40,9 @@ options {
 	pid-file "/run/named/named.pid";
 	session-keyfile "/run/named/session.key";
 
+	request-ixfr no;
+	allow-transfer { 1.1.1.1;};
```

Comment:
```suggestion
	allow-transfer { 1.1.1.1;};
	allow-recursion {"none";};
```

Reply:
Hey @Chandan-Abhyankar can you add this too? I found this one this weekend. Thanks, paul